pkg/upgrade/v1_18_0.go

```diff
@@ -0,0 +1,90 @@
+package upgrade
+
+import (
+	"context"
+	"fmt"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
this should be joined with the following block.

our import format tool seems to be flaky sometimes.